client/src/components/Navigation/Buttons/ButtonHome/ButtonHome.js

```diff
@@ -0,0 +1,25 @@
+import styled from 'styled-components/macro'
+import { IconContext } from 'react-icons'
+
+import { IoHomeSharp } from 'react-icons/io5'
+
+const ButtonHome = () => {
+  return (
+    <Wrapper>
+      <IconContext.Provider value={{ size: '30px' }}>
+        <IoHomeSharp />
+      </IconContext.Provider>
+      <span>Home</span>
+    </Wrapper>
+  )
+}
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 50;
+  width: 80;
+`
+
+export default ButtonHome
```

Comment:
I have a suggestion according to your structure of files: Won't it be better to have one button folder at the same level as the other components where you can save all buttons you need in your app? Therefore you won't need another button folder for the other components and the hierarchical structure would be flatter. 🤔